custom_typings/babel-7.d.ts

```diff
@@ -0,0 +1,9 @@
+// Babel 7 doesn't have typings yet. These are minimal and temporary
```

Comment:
*Still*?

Reply:
I've tried to `npm install @types/babel__{package-name}` for all of them.  There was a `@types/babel__template` but that still didn't even solve the compile issue...

```sh
> tsc

src/babel-plugin-dynamic-import-amd.ts:15:34 - error TS7016: Could not find a declaration file for module '@babel/plugin-syntax-dynamic-import'. '/Users/brendanb/src/github.com/Polymer/koa-esm-to-amd~pr1/node_modules/@babel/plugin-syntax-dynamic-import/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/babel__plugin-syntax-dynamic-import` if it exists or add a new declaration (.d.ts) file containing `declare module '@babel/plugin-syntax-dynamic-import';`

15 import dyanamicImportSyntax from '@babel/plugin-syntax-dynamic-import';
                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/babel-plugin-dynamic-import-amd.ts:16:9 - error TS2305: Module '"../../../../../../../Users/brendanb/src/github.com/Polymer/koa-esm-to-amd~pr1/node_modules/@types/babel__template"' has no exported member 'ast'.

16 import {ast} from '@babel/template';
           ~~~

src/babel-plugin-rewrite-import-meta.ts:15:30 - error TS7016: Could not find a declaration file for module '@babel/plugin-syntax-import-meta'. '/Users/brendanb/src/github.com/Polymer/koa-esm-to-amd~pr1/node_modules/@babel/plugin-syntax-import-meta/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/babel__plugin-syntax-import-meta` if it exists or add a new declaration (.d.ts) file containing `declare module '@babel/plugin-syntax-import-meta';`

15 import importMetaSyntax from '@babel/plugin-syntax-import-meta';
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/babel-plugin-rewrite-import-meta.ts:16:9 - error TS2305: Module '"../../../../../../../Users/brendanb/src/github.com/Polymer/koa-esm-to-amd~pr1/node_modules/@types/babel__template"' has no exported member 'ast'.

16 import {ast} from '@babel/template';
           ~~~
```